client/trino-client/src/main/java/io/trino/client/QueryResults.java

```diff
@@ -141,6 +142,7 @@ public List<Column> getColumns()
     @Nullable
     @JsonProperty
     @Override
+    @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
PR description suggests we're adding a test only, here we modify the core classes.
Is this "just a test" or "a fix with a test"?